workflows/README.md

```diff
@@ -1,2 +1,31 @@
-This contains all iwc worflows
-------------------------------
+# Workflows
+
+This directory contains all IWC workflows.
+
+The structure is as follows:
+
+* top-level directories represent categories, e.g., `sars-cov-2-variant-calling`;
+* directories right under the top level represent individual workflows, e.g., `sars-cov-2-consensus-from-variation`;
+* workflow directories contain:
+
+  * the `.ga` workflow file, e.g., `consensus-from-variation.ga`;
+  * a [Planemo](https://github.com/galaxyproject/planemo) test file, with the same name as the workflow file, but with a `.yml` extension, e.g., `consensus-from-variation.yml`;
+  * a `test-data` directory with the test data used by Planemo;
+  * a [Dockstore](https://dockstore.org) [metadata file](https://docs.dockstore.org/en/develop/getting-started/github-apps/github-apps.html#workflow-yml-file) named `.dockstore.yml`;
+  * a [Workflow Testing RO-Crate](https://crs4.github.io/life_monitor/workflow_testing_ro_crate) metadata file named `ro-crate-metadata.json`;
+  * a `README.md` and a `CHANGELOG.md` file.
+
+
+## RO-Crate Metadata
```

Comment:
Is it OK for you if we generate the RO-Crate in the github workflow that deploys the repos to https://github.com/iwc-workflows ? I want to keep the artefacts to check in as minimal as possible.

Reply:
Sure. I've removed the RO-Crate metadata files in 2476874